packages/slice-machine/pages/_app.tsx

```diff
@@ -20,30 +20,14 @@ import "react-datepicker/dist/react-datepicker.css";
 import "src/css/modal.css";
 import "src/css/tabs.css";
 
-import { AppPayload, ServerState } from "@lib/models/server/ServerState";
+import { ServerState } from "@lib/models/server/ServerState";
 import ServerError from "@lib/models/server/ServerError";
-import { Library } from "@lib/models/common/Library";
 import Head from "next/head";
 
 async function fetcher(url: string): Promise<any> {
```

Comment:
only used once, not sure we need it.

Reply:
You mean we don't need to declare a function for it ? 

We had the same discussion for the loginModal and we choose to create an external function. This case is the same, so we should use the same tech strat, to stay coherent. If we want to remove the function in this case, we should also change the LoginModal.

What do you want to do ?